imports/wasi_snapshot_preview1/fs_test.go

```diff
@@ -1196,9 +1265,34 @@ func Test_fdReaddir(t *testing.T) {
 					},
 				}
 			},
-			buf: 0, bufLen: 27, // length is long enough for one more.
-			cookie:          3,  // previous d_next
-			expectedBufused: 27, // length to read exactly one more.
+			buf: 0, bufLen: 27, // length is long enough for exactly third.
+			cookie:          2,  // d_next of second.
```

Comment:
I have changed cookie 3 to 2, because when you send in cookie 2, you expect the third entry.